juntagrico/templates/cancelsubscription.html

```diff
@@ -23,16 +23,16 @@ <h3>
                 {% csrf_token %}
                 <input name="end_date" value="{{ end_date |date:"Y-m-d" }}" type="hidden"/>
                 <div class="form-group row">
-                    <label class="col-md-3" for="message">
+                    <label class="col-md-2" for="message">
```

Comment:
why change this? this should be consisten throughout the whole code

Reply:
As there is only one textarea in that form its label looks lost on the left. Maybe the textarea should be below the label in this case.
![imagen](https://user-images.githubusercontent.com/1621900/59163331-f4ced500-8aff-11e9-9a46-ff014c23c3f4.png)

Reply:
i see what you mean. its all not really a thing of beauty. i guess if we find a good looking solution we can make an exception here